### DISCUSSION TITLE
Submission not showing metrics

### DISCUSSION THREAD

**Discussion by vigsivan** [facebookresearch/fastMRI#261] [Q&A]
Hello, my most recent submission to the fastMRI leaderboard was successful, but I do not see metrics such as SSIM show up against the submission. For reference, my submission has the title LW1.

Please let me know if there is additional information that I can provide.

Thanks!
Vignesh

**Comment by mmuckley** [comment]
Hello @vigsivan, your submission returned null values for all the metrics. Typically this occurs due to some numerical issue with how you prepared the submission. You can use the validation data and the standard model pipeline to debug locally what those might be for your case. You could also compare your pixel-level outputs to one of the pretrained model outputs that we released, see [here](https://github.com/facebookresearch/fastMRI/blob/main/fastmri_examples/varnet/run_pretrained_varnet_inference.py).
